src/daemon/Greeter.cpp

```diff
@@ -261,6 +285,14 @@ namespace SDDM {
             qDebug() << "Greeter session failed to start";
     }
 
+    void Greeter::onDisplayServerReady(const QString &displayName)
```

Comment:
Can you talk me through this.

The greeter starts, the helper creates an X server, it gets a name, which we're sending back to the daemon.
Why does the daemon need to know this? For xauth cookie handling?

Reply:
i just wanted to know the display name from the daemon just like before, in case it's useful in the future.

Reply:
i guess that in the we can think about it in the future, if we really need it and aim for semplicity here

Reply:
the only place where it seems it's used is in Display::displayServerStarted() when the socket server is started but we might use something else, maybe a random name.
SocketServer creates the socket like this:
```
QString socketName = QStringLiteral("sddm-%1-%2").arg(displayName).arg(generateName(6));
```
Since the socket name is passed around and not guessed we can really drop the displayName from the socket name and we are good to go since we already use a generated name